app/renderer/components/Inspector/SessionInfo.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import { Table, Card } from 'antd';
+import SessionCodeBox from './SessionCodeBox';
+import { withTranslation } from '../../util';
+
+class SessionInfo extends Component {
+
+  componentDidMount () {
+    const { isIOS, isAndroid } = this.props.driver.client;
+
+    this.props.getActiveAppId(isIOS, isAndroid);
+    this.props.getServerStatus();
+  }
+
+  generateSessionTime () {
+    const { sessionStartTime } = this.props;
+
+    let time = Date.now() - sessionStartTime;
+    let minutes = Math.floor(time / 60000);
+
+    return minutes + ' minutes';
+  }
+
+  createTable (tableValues, keyName) {
+    const keyValue = keyName + 'Value';
+    const capsDataSource = tableValues.map(([name, value]) => ({key: name, [keyName]: name, [keyValue]: value }));
+
+    const capsColumns = [{
+      dataIndex: keyName,
+      key: keyName,
+    }, {
+      dataIndex: keyValue,
+      key: keyValue,
+    }];
+
+    return <Card>
+      <Table
+        columns={capsColumns}
+        dataSource={capsDataSource}
+        pagination={false}
+        showHeader={false}
+        scroll={{ y: 105, x: 100 }}
```

Comment:
magic numbers?

Reply:
yeah these should be made constants at the top of the file with a name that indicates their meaning and purpose